src/data/gladiatorData.js

```diff
@@ -1,6 +1,24 @@
+
+
+export const Dimachaerus = "Dimachaerus";
+export const Secutor = "Secutor";
+export const Retiarius = "Retiarius";
+
+/**
+ * Gladiator Type
+ * @enum {number}
+ * 
+ */
+export const GladiatorType = {
```

Comment:
This is a handy convention to use across modules that plugs into intellisense nicely in the IDE and gives a better inline definition of what's going on